app/helpers/application_helper/listnav.rb

```diff
@@ -73,7 +73,7 @@ def render_listnav_filename
       elsif @explorer
         "explorer"
       elsif @layout == "timeline"
```

Comment:
@skateman will `@layout` ever be timeline,  this` elsif` probably should go away too

Reply:
@h-kataria yes, we need it for non-explorer screens that have `Monitoring -> Timelines` capability. For example infra/hosts...